lib/cc/cli/init.rb

```diff
@@ -9,6 +9,7 @@ class Init < Command
 
       def run
         if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
+          create_default_configs
           fatal "Config file .codeclimate.yml already present.\nTry running 'validate-config' to check configuration."
```

Comment:
Yeah, if this default configs is the desired behavior, I think it should be a successful output exit code & message. I would suggest outputting the informational message about not changing `.codeclimate.yml` first, and then generate the default configs.

This can & probably should actually be refactored to be part of `#create_codeclimate_yaml` behavior. Putting it here make behavior kind of strange in that `codeclimate init` will not touch `.codeclimate.yml`, but will touch default engine configs, whereas `codeclimate init --upgrade` will refuse to generate default engine configs if `.codeclimate.yml` already has engines.